index-handlers/src/main/java/no/sikt/nva/nvi/index/apigateway/FetchInstitutionReportHandler.java

```diff
@@ -1,43 +1,73 @@
 package no.sikt.nva.nvi.index.apigateway;
 
+import static nva.commons.core.attempt.Try.attempt;
 import com.amazonaws.services.lambda.runtime.Context;
 import com.google.common.net.MediaType;
 import java.net.HttpURLConnection;
+import java.time.Year;
 import java.util.List;
-import no.sikt.nva.nvi.index.xlsx.ExcelWorkbookGenerator;
+import no.sikt.nva.nvi.index.aws.OpenSearchClient;
+import no.sikt.nva.nvi.index.aws.SearchClient;
+import no.sikt.nva.nvi.index.model.document.NviCandidateIndexDocument;
+import no.sikt.nva.nvi.index.utils.InstitutionReportGenerator;
 import nva.commons.apigateway.AccessRight;
 import nva.commons.apigateway.ApiGatewayHandler;
 import nva.commons.apigateway.RequestInfo;
 import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
 import nva.commons.apigateway.exceptions.UnauthorizedException;
+import nva.commons.core.JacocoGenerated;
+import org.slf4j.Logger;
 
 public class FetchInstitutionReportHandler extends ApiGatewayHandler<Void, String> {
 
+    private static final Logger logger = org.slf4j.LoggerFactory.getLogger(FetchInstitutionReportHandler.class);
+    private static final String PATH_PARAMETER_YEAR = "year";
+    private final SearchClient<NviCandidateIndexDocument> searchClient;
+
+    @JacocoGenerated
     public FetchInstitutionReportHandler() {
+        this(OpenSearchClient.defaultOpenSearchClient());
+    }
+
+    public FetchInstitutionReportHandler(SearchClient<NviCandidateIndexDocument> searchClient) {
         super(Void.class);
+        this.searchClient = searchClient;
     }
 
     @Override
     protected List<MediaType> listSupportedMediaTypes() {
-        return List.of(MediaType.MICROSOFT_EXCEL, MediaType.OOXML_SHEET);
+        // The first media type in the list is the default media type if the client does not specify a media type
+        // See RestRequestHandler#defaultResponseContentTypeWhenNotSpecifiedByClientRequest
+        return List.of(MediaType.OOXML_SHEET, MediaType.MICROSOFT_EXCEL);
```

Comment:
Found a bug for `MediaType.MICROSOFT_EXCEL` when i tested the endpoint, therefore setting `MediaType.OOXML_SHEET` as default and will fix bug in new PR